layouts/shortcodes/discordInvite.html

```diff
@@ -0,0 +1,25 @@
+<!--
+Shortcode for Discord Invite
+Usage:
+< discordInvite invite_code=<> size=<> color=<> image=<> >
+ -->
+
+<!-- #5865F2 is discord's offical "blurple" color -->
+<!-- Large, medium, small breakpoints are arbitrary. Currently, just using tailwind width classes, but this can be tweaked however needed. -->
+<a {{ with .Get "invite_code" }} href="https://discord.gg/{{ . }}" {{ end }} >
+<img src="/img/Discord-Logo+Wordmark-White.png"
+{{ if .Get "size" | eq "large"}}
```

Comment:
I think you can do:

```html
{{ size := .Get "size" }}
```
And use that as a variable for the below parts.

Reply:
Updated in latest commit.